Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -471,15 +471,15 @@ You can retrieve it yourself with your browser or use the [Apple App Site Associ
 }
 ```
 
-The "details" key inside "applinks" contains a JSON representation of an array that might contain one or more apps. The "appID" should match the “application-identifier” key from the app’s entitlements. Next, using the "paths" key, the developers can specify certain paths to be handled on a per app basis. Some apps, like Telegram use a standalone * (`"paths": ["*"]`) in order to allow all possible paths. Only if specific areas of the website should **not** be handled by some app, the developer can restrict access by excluding them by prepending a `"NOT "` (note the whitespace after the T) to the corresponding path. Also remember that the system will look for matches by following the order of the dictionaries in the array (first match wins).
+The "details" key inside "applinks" contains a JSON representation of an array that might contain one or more apps. The "appID" should match the "application-identifier" key from the app’s entitlements. Next, using the "paths" key, the developers can specify certain paths to be handled on a per app basis. Some apps, like Telegram use a standalone * (`"paths": ["*"]`) in order to allow all possible paths. Only if specific areas of the website should **not** be handled by some app, the developer can restrict access by excluding them by prepending a `"NOT "` (note the whitespace after the T) to the corresponding path. Also remember that the system will look for matches by following the order of the dictionaries in the array (first match wins).
 
 This path exclusion mechanism is not to be seen as a security feature but rather as a filter that developer might use to specify which apps open which links. By default, iOS does not open any unverified links.
 
 Remember that universal links verification occurs at installation time. iOS retrieves the AASA file for the declared domains (`applinks`) in its `com.apple.developer.associated-domains` entitlement. iOS will refuse to open those links if the verification did not succeed. Some reasons to fail verification might include:
 
 - The AASA file is not served over HTTPS.
 - The AASA is not available.
-- The `appID`s do not math (this would be the case of a *malicious* app. iOS would successfully prevent any possible hijacking attacks
+- The `appID`s do not math (this would be the case of a *malicious* app). iOS would successfully prevent any possible hijacking attacks.
```

Comment:
Thanks! I can totally encourage you to start using "GitHub's ["Suggested Changes"](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) feature (which can be quickly accepted and commited by simply clicking on the "Commit suggestion" button). You'll also get credit for the suggestion and become co-author of the commit.

```suggestion
- The `appID`s do not match (this would be the case of a *malicious* app). iOS would successfully prevent any possible hijacking attacks.
```